sdk/containerregistry/Azure.Containers.ContainerRegistry/src/ContainerRegistryClientOptions.cs

```diff
@@ -13,6 +13,17 @@ public class ContainerRegistryClientOptions : ClientOptions
     {
         internal string Version { get; }
 
+        /// <summary>
+        /// Gets or sets the authentication scope to use for authentication with AAD.
+        /// This defaults to the Azure Resource Manager "Azure Global" scope.  To
+        /// connect to a different cloud, set this value to "&lt;resource-id&gt;/.default",
+        /// where &lt;resource-id&gt; is one of the Resource IDs listed at
+        /// https://docs.microsoft.com/azure/active-directory/managed-identities-azure-resources/services-support-managed-identities#azure-resource-manager.
+        /// For example, to connect to the Azure Germany cloud, create a client with
+        /// <see cref="AuthenticationScope"/> set to "https://management.microsoftazure.de/.default".
+        /// </summary>
+        public string AuthenticationScope { get; set; } = "https://management.azure.com/.default";
```

Comment:
Why are we defaulting to the ARM scope vs. the ACR scope? Is the latter not available yet? Also, when both are available do we need to give guidance on when to add one vs. the other?

Reply:
The ACR scope is not currently available - we've requested that it be available by the time the service GAs.  When both are available, yes, we'll need to give guidance on when to use which.  We'll want to expose that differently, and it will take some discussion to understand how the two permissioning models are different.

Reply:
Chatted offline with Scott about this - we'll take this approach in Beta3 but revisit before GA.  An ideal solution would be (in order of preference)
- General ACR scope would make it unnecessary for customer to specify cloud
- Arch board guidance would be finalized and we would allow users to specify cloud in a Track 2 consistent way
- If the service provides two permissions models with different scopes associated with them, we would provide an API that communicates the differences between the two models, and either guides the user to choose the appropriate scope, or specifies it for them.

Reply:
Tracking with: https://github.com/Azure/azure-sdk-for-net/issues/21603